### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by pschaeffer** [validator/validator#2137]
### Summary

Should I use vnu.jar or jrt-fs.jar? What should be imported for Java? I doing testing under Windows and production under Linux.

Thank you

Peter Schaeffer

### URL for document that reproduces the problem

https://example.com/

### HTML/CSS/SVG snippet that reproduces the problem

```html
Should I use vnu.jar or jrt-fs.jar? What should be imported for Java? I doing testing under Windows and production under Linux.

Thank you

Peter Schaeffer
```

### Checker error messages

```shell
Should I use vnu.jar or jrt-fs.jar? What should be imported for Java? I doing testing under Windows and production under Linux.

Thank you

Peter Schaeffer
```

### Steps to reproduce

1.
2.
3.

### Expected behavior

Should I use vnu.jar or jrt-fs.jar? What should be imported for Java? I doing testing under Windows and production under Linux.

Thank you

Peter Schaeffer

### Actual behavior

Should I use vnu.jar or jrt-fs.jar? What should be imported for Java? I doing testing under Windows and production under Linux.

Thank you

Peter Schaeffer

**Comment by sideshowbarker** [comment]
Use vnu.jar. I don’t know what jrt-fs.jar is — but whatever it is, is has nothing do with using the HTML checker.

**Comment by pschaeffer** [comment]
Hello,

 

The Jar file, jrt-fs.jar is in vnu-runtime-image\lib. 

 

I got

 

SLF4J: Class path contains multiple SLF4J bindings.

SLF4J: Found binding in [jar:file:/C:/Users/pscha/HeadlampJetty/slf4j-1.7.32/slf4j-log4j12-1.7.32.jar!/org/slf4j/impl/StaticLoggerBinder.class]

SLF4J: Found binding in [jar:file:/C:/Users/pscha/HeadlampJetty/vnu/vnu.jar!/org/slf4j/impl/StaticLoggerBinder.class]

SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.

SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]

 

I took the binder class out of the jar file and the problem went away.

 

Thank you

 

Peter Schaeffer
